src/styles/auto-prefix.js

```diff
@@ -44,6 +55,8 @@ export default {
   set(style, key, value) {
```

Comment:
I was going to have you change the implementation of this `set` method but this is truly a pointless method. We really just need to remove all the uses of it and get rid of the method. We can do that in a separate PR.